Core/src/Propagator/StraightLineStepper.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace Acts {
 
-std::tuple<BoundTrackParameters, BoundMatrix, double>
+Result<std::tuple<BoundTrackParameters, BoundMatrix, double>>
```

Comment:
I will harmonize the naming of these in my following PR, and change from `double` to `ActsScalar`